lib/email_address/email_address_type.rb

```diff
@@ -33,16 +33,16 @@ class EmailAddress::EmailAddressType < ActiveRecord::Type::Value
 
   # From user input, setter
   def cast(value)
-    super(EmailAddress.normal(value))
+    super(self.class.parent.normal(value))
   end
 
   # From a database value
   def deserialize(value)
-    value && EmailAddress.normal(value)
+    value && self.class.parent.normal(value)
   end
   #
   # To a database value (string)
   def serialize(value)
-    value && EmailAddress.normal(value)
+    value && self.class.parent.normal(value)
```

Comment:
From where `parent` method? From `ActiveSupport` because there is `activerecord` requirement for test suits? That's incorrect. Also, I think, we should move `require 'active_record'` from `test/test_helper.rb` to specific tests, related to AR.

I mean, this will raise an exception in regular usage without AR, you can try.